protocols/netty/src/main/java/org/apache/james/protocols/netty/BasicChannelUpstreamHandler.java

```diff
@@ -234,7 +234,7 @@ public void exceptionCaught(ChannelHandlerContext ctx, ExceptionEvent e) throws
                     }
                     transport.writeResponse(Response.DISCONNECT, session);
                 }
-                LOGGER.error("Unable to process request", e.getCause());
+                LOGGER.info("Unable to process request", e.getCause());

```

Comment:
As discussed, maybe we could check that the exception is relative to a close channel exception to know that we want to log it only at info level.